src/win32.mak

```diff
@@ -117,33 +102,24 @@ check-host-dc:
 debdmd: check-host-dc debdmd-make
 
 debdmd-make:
-	$(DMDMAKE) "DDEBUG=-debug -g -unittest" "DOPT=" $(TARGETEXE)
+	$(DMDMAKE) "ENABLE_DEBUG=1" "DFLAGS=-unittest" $(TARGETEXE)
 
 reldmd: check-host-dc reldmd-make
 
 reldmd-make: $(GEN)\build.exe
-	$(RUN_BUILD) "DDEBUG=" "ENABLE_RELEASE=1" $(TARGETEXE)
+	$(RUN_BUILD) "ENABLE_RELEASE=1" $(TARGETEXE)
 
+# Not ENABLE_RELEASE to avoid -inline
 profile:
-	$(DMDMAKE) "DDEBUG=" "DOPT=-O -release -profile" $(TARGETEXE)
+	$(DMDMAKE) "ENABLE_PROFILE=1" "DFLAGS=-O -release" $(TARGETEXE)
 
-trace:
-	$(DMDMAKE) "DDEBUG=-debug -g -unittest" "DOPT=" $(TARGETEXE)
+trace: debdmd-make
```

Comment:
This target seems to be completely unnecessary, so I'd be okay with removing it.

Reply:
It's probably unused but I don't mind keeping it as a simple alias.